tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -391,33 +470,100 @@ def get_fd_counts():
                 backoff_sec=5)
             self.logger.info(f"Post-restart elections done.")
 
-            for node_name, file_count in get_fd_counts().items():
+            inter_restart_check()
+
+            for node_name, file_count in self._get_fd_counts().items():
                 self.logger.info(
                     f"Open files after {i} restarts on {node_name}: {file_count}"
                 )
 
-        # With increased overhead from all those segment rolls during restart,
-        # check that consume still works.
-        self._consume_all(topic_names, msg_count_per_topic,
-                          expect_transmit_time)
-
+    def _write_and_random_read(self, scale: ScaleParameters, topic_names):
+        """
+        This is a relatively low intensity test, that covers random
+        and sequential reads & validates correctness of offsets in the
+        partitions written to.
+
+        It answers the question "is the cluster basically working properly"?  Before
+        we move on to more stressful testing, and in the process ensures there
+        is enough data in the system that we aren't in the "everything fits in
+        memory" regime.
+
+        Note: run this before other workloads, so that kgo-verifier's random
+        readers are able to validate most of what they read (otherwise they
+        will mostly be reading data written by a different workload, which
+        drives traffic but is a less strict test because it can't validate
+        the offsets of those messages)
+        """
         # Now that we've tested basic ability to form consensus and survive some
         # restarts, move on to a more general stress test.
         self.logger.info("Entering traffic stress test")
         target_topic = topic_names[0]
 
-        stress_msg_size = 32768
+        # Assume fetches will be 10MB, the franz-go default
+        fetch_mb_per_partition = 10 * 1024 * 1024
```

Comment:
Either the name is wrong or the number is wrong, because it's not 10 million MBs, right? 

I believe it is the name, should be `fetch_bytes_ ...` ?